core/src/test/java/org/elasticsearch/index/mapper/RangeFieldMapperTests.java

```diff
@@ -245,6 +245,11 @@ public void doTestCoerce(String type) throws IOException {
         IndexableField pointField = fields[1];
         assertEquals(2, pointField.fieldType().pointDimensionCount());
 
+        // date_range ignores the coerce parameter and epoch_millis date format truncates floats (see issue: #14641)
+        if (type.equals("date_range")) {
+            return;
```

Comment:
nit: maybe just personal preference, but early returns in test look strange to me. Can you change this to execute the rest of the test only for `type.equals("date_range") == false`)